pyproject.toml

```diff
@@ -30,8 +30,8 @@ classifiers = [
 ]
 requires-python = ">=3.9"
 dependencies = [
-    "numpy>=1.22.0; python_version <= '3.11'",
-    "numpy>=1.26.0; python_version >= '3.12'",
+    "numpy>=1.22.0, <2.0.0; python_version <= '3.11'",
```

Comment:
What is status on 2.0 support?

Reply:
I think pynwb recently added it, right? I added it to most of spikeinterface but just being cautios here.

Reply:
I thought I heard they fixed it